src/gpl/src/replace.cpp

```diff
@@ -296,6 +297,29 @@ void Replace::doInitialPlace()
   ip_->doBicgstabPlace();
 }
 
+void Replace::runMBFF(int max_sz, float alpha, float beta, int threads) {
+
+  int num_flops = 0, num_paths = 0;
+  vector<odb::Point> points; 
+  vector<pair<int, int> > paths;
+
+  auto block = db_->getChip()->getBlock();
+  for (auto inst : block->getInsts()) {
+    auto mstr = inst->getMaster();
+    if (mstr->isSequential()) {
```

Comment:
Pass by const reference
```suggestion
  for (const auto& inst : block->getInsts()) {
    if (inst->getMaster()->isSequential()) {
```